types/netease-captcha/netease-captcha-tests.ts

```diff
@@ -0,0 +1,20 @@
+const config: NeteaseCaptcha.Config = {
+        captchaId: 'FAKE ID',
+        element: '#captcha',
+        mode: 'popup',
+        protocol: 'https',
+        width: '200px',
+        lang: 'en',
+        onVerify: (error: any, data?: NeteaseCaptcha.Data) => {
+            console.log(error, data);
+        }
+    };
+
+const onLoad: NeteaseCaptcha.onLoad = (instance: NeteaseCaptcha.Instance) => {
+    instance.refresh();
+    instance.destroy();
+};
+
+function init(initNECaptcha: NeteaseCaptcha.InitFunction): void {
```

Comment:
I think you might be able to just test this functionality without the `init` wrapping function:


```
window.initNECaptcha(config, onLoad)
```